newIDE/app/src/UI/ResponsivePaperOrDrawer.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import * as React from 'react';
+import Paper from './Paper';
+import { useResponsiveWindowWidth } from './Reponsive/ResponsiveWindowMeasurer';
+import Drawer from '@material-ui/core/Drawer';
+
+const styles = {
+  paper: {
+    display: 'flex',
+    width: 250,
+  },
+  drawerPaper: {
+    display: 'flex',
+  },
+};
+
+const drawerPaperProps = {
+  style: styles.drawerPaper,
+};
+
+const drawerModalProps = {
+  keepMounted: true,
+};
+
+/**
+ * Display a Paper element, for medium/large screens, or a Drawer on small screens.
+ */
+export const ResponsivePaperOrDrawer = ({
```

Comment:
really useful component.
I'm wondering if we could re-use it for the homepage menu too.
Not priority and maybe not straightforward though